test/unit/resources/postgres_hba_conf_test.rb

```diff
@@ -1,7 +1,7 @@
 # encoding: utf-8
 # copyright: 2017
 # author: Aaron Lippold, lippold@gmail.com
-# author: Rony Xavier, rx294@nyu.edu 
+# author: Rony Xavier, rx294@nyu.edu
```

Comment:
Hmm...not sure how I feel about this. I'd like to remove the line, but it is an external contributor.

Reply:
All this is doing is just fixing the whitespace.

I'd actually like to nuke all the author lines across the board from the whole chef-client codebase.  I never add them anymore for myself.

Reply:
Yeah, I was referring to the fact that usually we remove these lines...but not sure in this case due to it being an external contributor.